source/renderer/app/components/staking/delegation-center/DelegationCenterHeader.js

```diff
@@ -46,6 +47,12 @@ export default class DelegationCenterHeader extends Component<Props> {
     intl: intlShape.isRequired,
   };
 
+  with2Decimals = (value: number) => {
```

Comment:
@tomy1610 Can you move this function outside of react component?
It should be defined as a `helper`, not a member method of react component.